src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.richard;
+
+/**
+ * Created by Administrator on 8/25/2017.
+ */
+public class FizzBuzz {
+
+    /**
+     * @param number is the number.
+     * @return boolean value.
+     */
+    public boolean isFizz(int number) {
+        return number % 3 == 0 || String.valueOf(number).contains("3");
```

Comment:
Please add constant for 3 and 5
also for Fizz and Buzz

Reply:
Okay, I'll solve that problem.